.github/workflows/checks.yml

```
@@ -58,6 +58,14 @@ jobs:
         uses: ./.github/actions/setup
       - name: Transpile to upgradeable
         run: bash scripts/upgradeable/transpile.sh
+      - name: Check out copy of vanilla
+        uses: actions/checkout@v4
+        with:
+          path: .vanilla
+      - name: Install vanilla as dependency
+        run: |
+          mkdir -p node_modules/@openzeppelin
+          ln -sr ".vanilla/contracts" node_modules/@openzeppelin/contracts
```

Given that the contracts are already checked out, would it be simpler/faster to do 
```suggestion
      - name: Copy non-upgradeable contracts as dependency
        run:
          cp -r contracts node_modules/@openzeppelin/contracts
```